### ISSUE TITLE
Is there a way to insert images

### ISSUE THREAD

**Issue by ataylor09** [brunurd/RTFExporter#4]
I'm not even sure if RTF allows this, but is there a way to create images?

**Comment by leonaaraujo** [comment]
yeah, there is some possibility in RTF, as in the Pictures section of this doc: http://www.biblioscape.com/rtf15_spec.htm
but nothing in the RTFExporter side... I will keep this issue open as a feature request.
@ataylor09
